.github/workflows/catch_flaky.yml

```diff
@@ -0,0 +1,51 @@
+name: catch_flaky
+
+on:
+  pull_request:
+    paths:
+      - 'test/*/*.lua'
+
+jobs:
+  catch_flaky:
+    if: github.event_name == 'pull_request' &&
+        !contains(github.event.pull_request.labels.*.name, 'noflaky') &&
+        !contains(github.event.pull_request.labels.*.name, 'notest')
+
+    env:
+      TNT_RELEASE_PATH: /home/runner/tnt-release
+
+    runs-on: ubuntu-22.04
+    timeout-minutes: 180
+    steps:
+      - name: Install tarantool
+        uses: tarantool/setup-tarantool@v3
+        with:
+          tarantool-version: '3.5'
+
+      - name: Clone the module
+        uses: actions/checkout@v3
+        with:
+          # Fetch the entire history for all branches and tags. It is needed for
+          # upgrade testing.
+          fetch-depth: 0
+          # Enable recursive submodules checkout as test-run git module is used
+          # for running tests.
+          submodules: recursive
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Install test requirements
+        run: pip3 install --user -r test-run/requirements.txt
+
+      - name: Setup tt
+        run: |
+          curl -L https://tarantool.io/release/2/installer.sh | sudo bash
+          sudo apt install -y tt
+          tt version
+
+      - name: Setup luatest
+        run: tt rocks install luatest 1.2.1
```

Comment:
Why do we install some specific `luatest` version and not the latest? The same comment in another workflow